style.css

```diff
@@ -56,14 +58,188 @@
     text-align: center;
   }
 
-  .social-media {
+  .main-text .social-media {
     display: flex;
     align-items: center;
     justify-content: center;
     list-style: none;
   }
 
-  .social-media img {
+  .main-text .social-media img {
     margin-right: 20px;
   }
+
+  .main-text .disable-button {
+    display: flex;
+    justify-content: center;
+    margin: 20px;
+  }
+
+  .my-work-text {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    margin: 30%;
+  }
```

Comment:
- [ ] Kindly use [Inter](https://fonts.google.com/?query=inter) font for your text to make it look like the design

Reply:
resolved